spec/models/entities/sub_entities/organization_spec.rb

```diff
@@ -19,34 +19,35 @@
     describe '#map_to base Contact' do
       let(:connec_hash) {
         {
-          "name" => "Test Company",
-          "industry" => "ITC",
-          "email" => {
-            "address" => "test@test.com"
+          :name => "Test Company",
```

Comment:
Usually the input of the mapper is a hash with string keys because JSON.parse returns a hash with string keys. I'm okay to have a symbol keys hash for check that it works in real life